.github/workflows/ci-npm-publish.yml

```diff
@@ -0,0 +1,53 @@
+name: "NPM Publish"
+
+on:
+  release:
+    types: [created]
+
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    services:
+      mongo:
+        image: mongo:4.4
+        ports:
+          - 27017:27017
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: 12
+          registry-url: https://registry.npmjs.org/
+      # Skip post-install scripts here, as a malicious
+      # script could steal NODE_AUTH_TOKEN.
+      - name: Install NPM dependencies
+        run: npm install --ignore-scripts
+
+      - name: Run linting
+        run: npm run lint
+
+      - name: Run tests
+        run: npm run test-ci
+
+  publish-npm:
+    name: NPM Publish
+    needs: build
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: 12
+          registry-url: https://registry.npmjs.org/
+      # Skip post-install scripts here, as a malicious
+      # script could steal NODE_AUTH_TOKEN.
+      - name: Install NPM dependencies
+        run: npm install --ignore-scripts --production
+
+      - name: Run NPM publish
+        run: npm publish
```

Comment:
should be: `run: npm publish --access public`

Reply:
done